src/profile/external-identities/ExternalIdentities.tsx

```diff
@@ -12,23 +12,33 @@ type ExternalIdentitiesProps = {
 
 export const ExternalIdentities = (props: ExternalIdentitiesProps): ReactNode => {
   const { queryParams } = props;
+  const providers = getConfig().externalCreds?.providers;
+  const desiredOrder: OAuth2ProviderKey[] = [
+    'ras',
+    'fence',
+    'dcf-fence',
+    'kids-first',
+    'anvil',
+    'github',
+    'era-commons',
+  ];
+  const sortedProviders = providers.sort((a: any, b: any) => desiredOrder.indexOf(a) - desiredOrder.indexOf(b));
```

Comment:
Could you simplify this by changing the order of providers directly in the .conf files, e.g. https://github.com/DataBiosphere/terra-ui/blob/7c5332ec8c757356ffc4f240f30509c068daedcf/config/prod.json#L13

Reply:
Fixed: https://github.com/DataBiosphere/terra-ui/pull/5259/commits/d458c307f879248cf2011550260394d0630cbe34